server/JsDbg.Gdb/JsDbg.py

```diff
@@ -151,7 +151,10 @@ def GetAllFields(module, type, includeBaseTypes):
             # Field is static
             continue
 
-        
+        if not field.name:
+            # Field is anonymous, we have no way to access it.

```

Comment:
I'm curious...how do we get this information in the debugger (GDB)?

Reply:
Looks like you access it as if it were a direct member of LayoutBox (like in C++):
(gdb) p $lo
$3 = (blink::LayoutBlockFlow *) 0x3587f6e22ab8
(gdb) p $lo->inline_box_wrapper_
$4 = (blink::InlineBox *) 0x0

So we could probably just consider them part of the parent, as long as we're careful with the bit offset...

Reply:
OK, please hold off on merging this, I really need to resolve this -- blink::Length is pretty useless without supporting anonymous unions

Reply:
Ok, will un-approve for now so it stays on my list.

Reply:
Done -- please take another look!